cmd/tools_netem.go

```diff
@@ -249,6 +258,67 @@ func qdiscToTableData(qdisc gotc.Object) tableWriter.Row {
 	}
 }
 
+// ImpairmentData represents a simplified view of the impairments
+// for JSON output.
+type ImpairmentData struct {
+	Interface  string `json:"Interface"`
+	NodeName   string `json:"NodeName"`
```

Comment:
Perhaps it would be a better idea to structure the output like this?

```
{
 "<nodename>": [
  {
   "interface": "<interface_name>",
    "delay": ...
  }
 ]
}
 ```
 
 This way it's easier to retrieve a specific node's impairments, rather than having to filter on the value of the node_name attribute within an impairment.